config/routes.rb

```diff
@@ -12,6 +12,7 @@
     namespace :v1, defaults: { format: :json } do
       devise_scope :user do
         resources :topics, only: :index
+        resources :targets, only: %i[create index destroy]
```

Comment:
```suggestion
        resources :targets, only: :create
```

You should then change it to `resources :targets, only: %i[create index]` in the PR you add index logic & `        resources :targets, only: %i[create index destroy]` in the PR you add the destroy logic